docs/source/contents/4_draft_mapping.rst

```diff
@@ -2152,6 +2152,622 @@ Drop all values in mods:part.
+      <titleInfo>
+        <title>A. G. "Dutch" and Margaret Ann  Roth  Papers</title>
+      </titleInfo>
+      <identifier>MS.3334</identifier>
+    </relatedItem>
+    <relatedItem displayLabel="Project" type="host">
+      <titleInfo>
+        <title>Albert "Dutch" Roth Photograph Collection</title>
+      </titleInfo>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+    @prefix opaque: <http://opaquenamespace.org/ns/> .
+
+    <https://example.org/objects/1> dbo:collection """A. G. "Dutch" and Margaret Ann Roth Papers, MS.3334""" ;
+        opaque:memberOfArchivalSeries "Series II: Margaret Ann Roth Photographs and Other Materials, 1947 March 11-2002 December 14 (bulk 1947 March 11-1955 March 20). Sub-Series A: Photographs, 1947 March 11-1955 March 139" .
+
+relatedItem[@type='host'][@displayLabel='Bibliographic Citation']
+-----------------------------------------------------------------
+Use Case
+^^^^^^^^
+This XPath only appears 1264 times in the Arrowmont Collection, specifically the Arrow of Pi Beta Phi sub-collection.
+
+Justification
+^^^^^^^^^^^^^
+XPath
+^^^^^
+:code:`relatedItem[@type='host'][@displayLabel='Bibliographic Citation']`
+
+Decision
+^^^^^^^^
+The `dcterms:bibliographicCitation` predicate was selected for these values.
+
+`Example record - arrow:1 <https://digital.lib.utk.edu/collections/islandora/object/arrow:1/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <relatedItem type="host" displayLabel="Project">
+      <titleInfo>
+        <title>From Pi Beta Phi to Arrowmont</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem displayLabel="Project Part" type="host">
+      <titleInfo>
+        <title>The Arrow of Pi Beta Phi</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem displayLabel="Bibliographic Citation" type="host">
+      <titleInfo>
+        <title>The Arrow, Volume 27, Number 1</title>
+      </titleInfo>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:bibliographicCitation "The Arrow, Volume 27, Number 1" .
+
+relatedItem[@type="otherVersion"]
+---------------------------------
+Use Case
+^^^^^^^^
+`relatedItem[@type="otherVersion"]` is used to indicate identifying information about another version of the resource. It appears in the Van Vactor and Arrowmont metadata; it is used, respectively, to identify an alternate version of the sheet music or the scrapbook that holds the image.
+
+Justification
+^^^^^^^^^^^^^
+
+XPath
+^^^^^
+:code:`relatedItem[@type='otherVersion']/identifier` OR
+:code:`relatedItem[@type='otherVersion']/location/url`
+
+Decision
+^^^^^^^^
+See the following section on `relatedItem/identifier[@type]`.
+
+I'm not sure about the relevance of `relatedItem[@type='otherVersion']/location/url`. In theory, it would look something like this, but it would need to be added to our metadata/RDF after migration:
```

Comment:
I think this was added in arrowmont so that users could immediately see the context of the images (where it lies in the scrapbook/album). This text is present on the [meeting notes page for the migration](https://jirautk.atlassian.net/wiki/spaces/DLP/pages/2326636): "Search results would return individual images but there's a link under the image that would take the user to page of the scrapbook (title would also indicate what the image is a part of)". I think we need to rewrite what you have here to be more confident. I'd agree though that obviously the links themselves won't hold following a migration.

Reply:
@mlhale7 - thanks for linking to the Arrowmont migration project docs. In some ways this seems this (setting up links from an item to scrapbook) should be a post-migration project, but maybe there's a compelling reason to think about it as part of our migration?

Reply:
If you can slightly edit this to remove "I'm not sure of the relevance of" to be more conclusive, I'm happy to resolve this.